plugins/pencilblue/controllers/admin/content/articles/manage_articles.js

```diff
@@ -34,7 +34,7 @@ module.exports = function(pb) {
     ManageArticles.prototype.render = function(cb) {
         var self = this;
         if(!pb.security.isAuthorized(this.session, {logged_in: true, admin_level: pb.SecurityService.ACCESS_EDITOR})) {
-            where.author = this.session.authentication.user_id;
+            where.author = this.session.authentication.user_id; // Where is the object? Is this just a copy-paste?
```

Comment:
This one actually gave me an error today, stating `where is not defined` when viewing the exact page as a writer